### PR TITLE
[4.0][com_media] RTL: adapting media manager to rtl + some

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes
This PR corrects the rtl display of the media manager.
Plus:
It also modifies 2 language strings (dates) and allows Jalali date to be displayed.
It adds a missing lang string.
It adds to the frontend com_media.ini the missing lang strings to let create site only lang packs.
It orders language strings in the ini files as well as in default_texts.php

What it does not do:
It does not delete the frontend language strings which are no more used.
It does not solve the frontend infobar absent display.
It does not solve frontend Inserting image in article `TypeError: Joomla.getImage is not a function`

### Testing Instructions
Install the Persian Language pack (administrator/index.php?option=com_installer&view=languages)
Switch to Persian in administrator.
Display the media manager.

**Patch and run npm ci**

### Before patch
This is what we get for LTR

screen shot 2019-02-28 at 08 23 57ltr

This is what we get for RTL

screen shot 2019-02-28 at 08 19 20rtlbefore

### After patch
This is what we should get for RTL

screen shot 2019-02-28 at 12 09 33

### Supplementary test
After patch and npm ci
Enable Debug lang.
Delete or change name of `administrator/language/en-GB/en-GB.com_media.ini` in order to load ONLY the front end file.
Edit an article in front end and display the xtd Images modal
There should not be any more untranslated strings

### Documentation Changes Required
None

### CODE REVIEW

File: language/en-GB/en-GB.com_media.ini [hunk 1]
```diff
@@ -55,34 +69,54 @@ COM_MEDIA_FIELD_PATH_IMAGE_FOLDER_DESC="Enter the path to the image folder relat
...
 COM_MEDIA_TITLE="Image Title"
 COM_MEDIA_TOGGLE_INFO="Toggle item info"
 COM_MEDIA_TOGGLE_LIST_VIEW="Toggle between grid and table views"
 COM_MEDIA_TOGGLE_SELECT_ITEM="Toggle select item"
+COM_MEDIA_TOOLBAR_LABEL="Media Manager"
 COM_MEDIA_UP="Up"
 COM_MEDIA_UPLOAD="Upload"
 COM_MEDIA_UPLOAD_COMPLETE="Upload Complete"
 COM_MEDIA_UPLOAD_FILE="Upload file"
+COM_MEDIA_UPLOAD_SUCCESS="Item uploaded."
```
Comment (C-Lodder) [inline language/en-GB/en-GB.com_media.ini:121]: Do you need the fullstop (`.`) at the end?

Comment (infograf768) [reply language/en-GB/en-GB.com_media.ini:121]: I have not corrected various existing lang strings.
We do have already
`COM_MEDIA_DELETE_SUCCESS="Item deleted."` for example.
or
`COM_MEDIA_ERROR="An error occurred."`
I think such messages should end with a period.

As you can see the capitals and periods for these are not consequent.
I can though modify them on the model of
`COM_CATEGORIES_SAVE_SUCCESS="Category saved."` etc.

In this case, we should have I guess
```
COM_MEDIA_UPLOAD_COMPLETE="Upload complete."
COM_MEDIA_UPLOAD_FILE="Upload file"
COM_MEDIA_UPLOAD_SUCCESS="Item uploaded."
```

I guess these can be modified later on by specialists in a specific PR, to not interfere with this PR.
What do you think?

Comment (infograf768) [reply language/en-GB/en-GB.com_media.ini:121]: Looks also that 
`COM_MEDIA_UPLOAD_FILE` is not used at all in 4.0

These ini will have to be drastically cleaned up when media manager gets more stable.

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**